docs/Advanced-Usage/Chip-Communication.rst

```diff
@@ -0,0 +1,147 @@
+.. _chip-communication:
+
+Communicating with the Chip/DUT
+===============================
+
+What good is a chip/DUT if it can't communicate with the outside world? There are two types of designs that can be
+made: tethered or standalone designs. A tethered design is where a host must interact with the DUT (a target) to bringup the design.
+A standalone design is a design that can bringup itself (has its own bootrom, loads programs itself, etc). An example of a tethered design
+is a Chipyard simulation where the host computer loads the test program into the designs memory. An example of a standalone design is
+a design where a program can be loaded from an SDCard by default.
+
+Chipyard designs communicate to the outside world in one of two ways that mainly correspond to a tethered design:
```

Comment:
This sentence can be made clearer.

"There are two ways the outside world (or host) can communicate with a tethered Chipyard design"